tf_module/ckr_aws/main.tf

```diff
@@ -1,3 +1,7 @@
+terraform {
+  required_version = "< 0.15.0"
```

Comment:
Could you pop a comment on explaining that this is because of the terraform orb, not because of anything in the terraform itself? It'd be sad to come back to this in a few months time and be trying to work out why we didn't think this was 0.15 compatible

Reply:
Although I guess now we can just look back at my comment on this PR :laughing:

Reply:
thanks 👍 

added a comment in 8ba8f22